lib/rspec/core/reporter.rb

```diff
@@ -73,7 +73,10 @@ def start(expected_example_count, time=RSpec::Core::Time.now)
       notify :seed, Notifications::SeedNotification.new(@configuration.seed, seed_used?)
     end
 
-    # @private
+    # @api
```

Comment:
I think it's probably wrong there then too :).  A good way to check is to run `yard server --reload` and see what's rendered as you add and remove this tag.  In general, I try to always view docs I add to make sure they render well.